content/maxwell3_fdem/sphere_freespace.rst

```diff
@@ -0,0 +1,262 @@
+Here, expressions derived by Wait (1951) are used to characterize the excitation of the sphere in terms of an induced dipole moment.
+We examine how a target's dipole moment depends on its size and physical properties.
+A simple model for predicing the sphere's dipole response in free-space is then presented.
+For a sufficiently compact target, the free-space response represents a low-order approximation to the response of a conductive and permeable object within a resistive medium.
+
+
+Plane Wave Excitation of a Conducting and Magnetically Permeable Sphere
+=======================================================================
+
+In this section, equations derived by Wait (1951) are used to describe the induced magnetic dipole moment of a conductive and permeable sphere, resulting from excitation by a uniform magnetic field (Figure ??).
+According to Ward and Hohmann (1989), the frequency-domain wave equation within a homogeneous media can be expressed in terms of the magnetic Schelkunoff potential :math:`\vec F`:
+
+.. figure:: ./images/figGeometrySphere.png
+	:align: right
+        :scale: 45%
+        :name: GeometrySphere
+
+.. math::
+	\nabla^2 \vec F - \gamma^2 \vec F = 0
+	:label: Schelkunoff_Wave
+
+For a propagation media with conductivity :math:`\sigma`, magnetic permeability :math:`\mu` and electric permittivity :math:`\varepsilon`, the wavenumber :math:`\gamma` can be expressed as follows:
+
+.. math::
+	\gamma = \Big [ i \omega \mu \sigma - \omega^2 \mu \varepsilon \Big ]^{1/2}
+	:label: Wave_Number
+
+By definition of the Schelkunoff potential, the electric field :math:`\vec E` may be obtained by substituting :math:`\vec F` into the following expression:
+
+.. math::
+	\vec E = - \nabla \times \vec F
+	:label: EcurlF
+
+And for the magnetic field :math:`\vec H`:
+
+.. math::
+	\vec H = - \big (\sigma + i \omega \varepsilon \big ) \vec F + \frac{1}{i \omega \mu} \nabla \big ( \nabla  \cdot \vec F \big )
+	:label: SchelkunoffH
+
+For his derivation, Wait (1951) began by considering the induced magnetic dipole moment resulting from an incident plane wave.
+If the wavelength of the incident wave is sufficiently larger than the radius of the sphere (i.e. :math:`|\gamma_b |/2\pi \ll R`), then we may assume the magnetic field which induces the dipole moment is approximately uniform about the sphere.
+For an inducing field of the form :math:`\vec H_0 (i\omega) = H_0 e^{i\omega t} \hat z`, symmetry of the problem implies that :math:`\vec E` only has components in :math:`\hat \phi`.
+Therefore by Eq. :eq:`EcurlF`, it follows that our Schelkunoff potential will only have components in :math:`\hat z`.
+The Schulkunoff potential may be obtained by considering seperate solutions inside and outside of the sphere:
+
+.. math::
+	\vec F (\omega) = \begin{cases}
+	F_b e^{i \omega t} \hat z \; \; \textrm{  at  } \; \; r>R \\
```

Comment:
I find the notation _b for background is confusing as the word "background" does not appear anywhere.
(I see for the electrostatic sphere we used _2 for inside the sphere and _1 for inside. It might not be the best but it would be nice to be consistent, one way or the other)